src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/appointments/AppointmentsToNomisIntTest.kt

```diff
@@ -356,6 +356,9 @@ class AppointmentsToNomisIntTest : SqsIntegrationTestBase() {
           appointmentsApi.stubGetAppointmentInstance(id = APPOINTMENT_INSTANCE_ID, response = appointmentResponse)
 
           publishAppointmentEvent("appointments.appointment-instance.created")
+
+          await untilCallTo { appointmentsApi.getCountFor("/appointment-instances/$APPOINTMENT_INSTANCE_ID") } matches { it == 1 }
+          await untilCallTo { nomisApi.postCountFor("/appointments") } matches { it == 1 }
```

Comment:
This is needed to make the test work - something weird happens with Wiremock when trying to verify too early and you can't assert on the JSON.